### DISCUSSION TITLE
如果有js framework支持，就可以让一般网站也可以混合使用？现在有吗？

### DISCUSSION THREAD

**Discussion by punkcanyang** [IntensiveCoLearning/Web3-URL#88] [General]
有没有类似的套件呢？或是可以从什么方向去开发？

**Comment by qzhodl** [comment]
指的是这个吗：https://github.com/web3-protocol/web3protocol-js

**Reply by punkcanyang** [reply]
是的，我看他的源码是直接对着EVM去数据，也就是说透过[web3protocol-js](https://github.com/web3-protocol/web3protocol-js)应该可以不需要网关，结合到现在的Web模式，可以让一般的网页获取Web3URL上的资源
